### DISCUSSION TITLE
WalletConnect with Trust Wallet Core

### DISCUSSION THREAD

**Discussion by deletexl** [trustwallet/wallet-core#2980] [Q&A]
Hello, I apologize for my bad English, I hope I can express myself with the translation. 

As you know, the WalletConnect library is used as a bridge to connect dApps and wallets. Is such a thing possible with Trust Wallet Core? Can an android application developed with Trust Wallet Core interact with Trust Wallet and link the wallet to the application?
Can we connect android apps using Trust Wallet Core to the wallet?

**Comment by gastonm5** [comment]
Hello @deletexl! Wallet Core is not related to WalletConnect, but Trust Wallet supports it so you could connect your Dapp to the mobile app.
For more information and dev support, feel free to join our public developers group: https://t.me/trust_developers :)

**Reply by deletexl** [reply]
> Hello @deletexl! Wallet Core is not related to WalletConnect, but Trust Wallet supports it so you could connect your Dapp to the mobile app. For more information and dev support, feel free to join our public developers group: https://t.me/trust_developers :)

How I can connect my app the trust wallet with trust wallet core have any reference or example?

**Reply by gastonm5** [reply]
You should check [WalletConnect docs](https://docs.walletconnect.com/2.0/). They have integration examples

**Reply by deletexl** [reply]
Can we do this with Trust Wallet Core?
